detection-rules/callback_phishing_phone_number_in_subject.yml

```diff
@@ -0,0 +1,70 @@
+name: "Callback Phishing phone number in subject"
+description: |
+  This rule inspect messages with no attachments, from first-time senders,
+  that contain phone number patterns in the subject and specific keywords associated
+  with callback phishing brands.
```

Comment:
```suggestion
  Detects phone number patterns in the subject and keywords associated with callback phishing brands from first-time senders.
```

Reply:
i think it helps to have the more pertinent / high signal stuff mentioned first in descriptions